src/contracts/__tests__/tasks.test.ts

```diff
@@ -0,0 +1,42 @@
+import { request } from 'contracts/request'
+import { createTask, deleteTask, getTaskById, updateTask } from 'contracts/tasks'
+import { task } from '__mocks__/entities/task.mock'
+
+jest.mock('contracts/request', () => ({
+  request: {
+    get: jest.fn(),
+    put: jest.fn(),
+    delete: jest.fn(),
+    post: jest.fn()
+  }
+}))
+
+const testUrl = `tasks/${task.id}`
+
+describe('getTaskById', () => {
+  test('should return data on request', () => {
+    getTaskById(task.id)
+    expect(request.get).toHaveBeenCalledWith(testUrl)
+  })
+})
+
+describe('updateTask', () => {
+  test('should update data on request "updateTask"', () => {
```

Comment:
```suggestion
  test('should call "request.put()" on "updateTask"', () => {
```
describe what you're doing in the test